security/src/main/scala/com/example/auction/utils/JsonFormats.scala

```diff
@@ -3,7 +3,7 @@ package com.example.auction.utils
 import java.time.Duration
 import java.util.UUID
 
-import play.api.data.validation.ValidationError
+import play.api.libs.json.JsonValidationError
```

Comment:
This was probably an update in Play 2.6.x

Reply:
It's missing from the Play 2.6 migration guide as well. I raised https://github.com/playframework/playframework/issues/8190.

I think it's still worth highlighting in ours, since Play JSON is used directly in Lagom Scala services.